readme.md

```diff
@@ -77,7 +77,7 @@ You are required to create an environment file (in JSON format) with the followi
 | CLIENT_ID |  Client id provided during onboarding (e.g. _STG2-MYINFO-SELF-TEST_)|
 | REDIRECT_URL | The callback URL specified when invoking the authorize call. For our sample application, it is http://localhost:3001/callback |
 | SCOPE | Space separated list of attributes to be retrieved from Myinfo.  |
-| AUTHORIZE_JWKS_URL | The callback URL specified when invoking the authorize call. For our sample application, it is http://localhost:3001/callback |
+| Callback_URL | The callback URL specified when invoking the authorize call. For our sample application, it is http://localhost:3001/callback |

```

Comment:
**suggestion:** should probably follow the same style as the rest by having it in all caps.